src/System.Management.Automation/engine/MshSnapinQualifiedName.cs

```diff
@@ -62,11 +64,11 @@ private PSSnapinQualifiedName(string[] splitName)
         /// <returns>
         /// An instance of the Name class.
         /// </returns>
-        internal static PSSnapinQualifiedName GetInstance(string name)
+        internal static PSSnapinQualifiedName? GetInstance(string name)
         {
             if (name == null)
```

Comment:
we can remove the null check `(name == null)`

Reply:
No, but I'll make name nullable